redirect.json

```
@@ -110,6 +110,20 @@
     "description": "show your devices",
     "introduced": "2021.3"
   },
+  {
+    "redirect": "config_energy",
+    "name": "Energy Configuration",
+    "badge": "Energy",
+    "description": "show your Energy configuration panel",
+    "introduced": "2021.8"
+  },
+  {
+    "redirect": "energy",
+    "name": "Energy Panel",
+    "badge": "Energy",
```

Differentiate the energy settings & energy badges?
```suggestion
    "badge": "Energy Settings",
```